### PR TITLE
Fix rubocop namespaces.

### PR DESCRIPTION
**Pull request by eugenk**
This is the same as https://github.com/ontohub/ontohub-backend/pull/126, only for this repository.

### CODE REVIEW

**Comment by eugenk** [comment]
How about now?

File: .rubocop_modified.yml [hunk 1]
```diff
@@ -1,19 +1,17 @@
 Metrics/BlockLength:
   Exclude:
-    - 'ontohub-models.gemspec'
```
Comment (phyrog) [inline .rubocop_modified.yml]: Are these removals intentional?

Comment (eugenk) [inline .rubocop_modified.yml]: no

File: .rubocop_modified.yml [hunk 2]
```diff
@@ -2,18 +2,17 @@ Metrics/BlockLength:
   Exclude:
     - 'ontohub-models.gemspec'
     - 'spec/**/*'
-    - 'db/migrate/*'
```
Comment (phyrog) [inline .rubocop_modified.yml]: And this?

**Pull request merged**